about.html

```diff
@@ -0,0 +1,199 @@
+            >
+          </p>
+          <img
+            src="./build/images/sprintplex-c-c.png"
+            alt=""
+            class="logo-description__2023logo"
+          />
+        </div>
+      </div>
+    </section>
+    <section class="past-editions section" id="speakers">
+      <div class="row past-editions__container">
+        <h2 class="past-editions__title section-title">
+          See past editions of the Coding Camp
+        </h2>
+        <div class="past-editions__items-container about-container ">
+          <div class="past-editions__video">
+            <video class="video-clip" width="320" height="240" controls>
+              <source src="movie.mp4" type="video/mp4">
+              <source src="movie.ogg" type="video/ogg">
+            Your browser does not support the video tag.
+            </video>
+          </div>
+          <div class="past-editions__video">
+            <video class="video-clip" width="320" height="240" controls>
+              <source src="movie.mp4" type="video/mp4">
+              <source src="movie.ogg" type="video/ogg">
+            Your browser does not support the video tag.
+            </video>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="partners section" id="partners">
+      <div class="row partners__container">
+        <h2 class="partners__title section-title">Partners</h2>
+        <div class="partners__list">
+          <img
+            src="./build/images/google-greyscale-removebg-preview.png"
+            alt=""
+            class="partners__item"
+          />
+          <img
+            src="./build/images/Workplace-Logo-removebg-preview.png"
+            alt=""
+            class="partners__item"
+          /><img
+            src="./build/images/cisco-greyscale-removebg-preview.png"
+            alt=""
+            class="partners__item"
+          />
+          <img
+            src="./build/images/linkedin-greyscale-removebg-preview.png"
+            alt=""
+            class="partners__item"
+          />
+          <img
+            src="./build/images/paypal-greyscale-removebg-preview.png"
+            alt=""
+            class="partners__item"
+          />
+        </div>
+      </div>
+    </section>
+    <footer class="footer" id="footer">
+      <div class="footer__container row">
+        <div class="footer__organizer">
+          <img
+            src="./build/images/sprintplex.jpeg"
+            alt=""
+            class="footer__organizer-logo"
+          />
+        </div>
+        <div class="footer__detail">
+          <p class="footer__copyright">
+            &copy;2022 Sprintplex Limited All rights reserved
+          </p>
+          <p class="footer__copy">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Doloremque
+            molestias rerum eius saepe facere fugit labore perspiciatis odit
+            veniam dolor.
+          </p>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] As you can see in the design the footer of the about page has a dark background. Please kindly fix that 🙃 😉

Your design
<img width="1021" alt="Screen Shot 2022-12-02 at 02 12 45" src="https://user-images.githubusercontent.com/74501523/205178801-9c033ab8-45cd-4889-8ad4-4a4b34eca955.png">

Origin design
<img width="874" alt="Screen Shot 2022-12-02 at 02 13 06" src="https://user-images.githubusercontent.com/74501523/205178885-2fa2da81-0ca8-453e-ae5e-7d7cf9700e5b.png">